### PR TITLE
feature/Android-SplashScreen PR

### LINKED ISSUE #5: features/Android-SplashScreen
* Implement the splash screen design for iPhones
* Test it for different screen sizes and make sure the dimension ratio is not skewed 
* Test it on multiple iPhone simulators and share the screenshots in the PR description
* Make sure you use auto-layout 

Estimated Time: 6hr
Time taken:

### PR DESCRIPTION
**Pull request by usaidam1n**
**Implementation:**
* Installed react-native-splash-screen package.
* Added splash-screen logo for android and implemented splash-screen.
* Set color of the app as discussed #272727
* Imported SplashScreen on App.tsx in order to hide the splash-screen once the content loads.

**Testing:**
* Tested my changes on OnePlus 8 and Google nexus S[480x800].
* Splash screen was showing on both devices.

___
**Screenshot/Video:**
OnePlus 8: 
IMAGE 2021-09-09 15:25:52

https://user-images.githubusercontent.com/46266811/132665312-adbb539d-a685-4644-ae2e-cf4f84570f51.mov

Google nexus S:
Screen Shot 2021-09-09 at 14 38 57

https://user-images.githubusercontent.com/46266811/132665626-2dc87b91-cfee-48b8-aedc-736d818bdc40.mov

### CODE REVIEW

File: android/app/src/main/res/layout/launch_screen.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (saamerm) [inline android/app/src/main/res/layout/launch_screen.xml:1]: @usaidpeerzada Is this file needed and being used anywhere?

**Review by saamerm — CHANGES_REQUESTED** [review]

File: android/app/src/main/res/layout/launch_screen.xml (hunk 1 above)
Comment (usaidam1n) [reply android/app/src/main/res/layout/launch_screen.xml:1]: Yes, this is the file  react-native-splash-screen package reads  and this file needs to be exactly as named and in layout folder to work properly.

**Review by saamerm — APPROVED** [review]
